frontend/HelperFuncs/plaidHelp.js

```diff
@@ -4,11 +4,38 @@ File to provide suite of client-side helpers for integrating Plaid API
 
 
 export const fetchLinkToken = async (userId) => {
-    console.log("Fetching link token from server for userId:", userId);
-    const response = await fetch(`http://localhost:3000/api/create_link_token/${userId}`, { method: 'POST' });
+    const response = await fetch(`http://localhost:3000/api/create_link_token/${userId}`, 
+    { method: 'POST' });
     if (!response.ok) {
         throw new Error('Network response was not ok');
     }
     const data = await response.json();
     return data.link_token;
-};
\ No newline at end of file
+};
+
+
+export const fetchAccessToken = async (userId, public_token) => {
+    const response = await fetch(`http://localhost:3000/api/exchange_public_token/${userId}`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ public_token })
+    });
+    if (!response.ok) {
+      throw new Error('Network response was not ok');
+    }
+    const data = await response.json();
+    return data.access_token;
+  };
+
+
+export const postTransactions = async(userId) => {
+    const response = await fetch(`http://localhost:3000/api/transactions/sync/${userId}`, 
+    { method: 'POST'} );
+    if(!response.ok) {
+        throw new Error('Network response was not ok');
```

Comment:
Would it be helpful for your debugging if you also added the error that cause the not ok response to the message?

Reply:
Yes, I think at scale it'd be better to specify. The reason the messages are pretty ambiguous right now is that once something has worked during testing it tends to stay working so debugging has been pretty linear thus far. But I agree for maintenance debugging messages should be more modular. I'll add that to my TODO list for my next PR